src/commands/messageCount.js

```diff
@@ -0,0 +1,72 @@
+import {
+  isCountUserMessage,
+  addCountUserMessage,
+  removeCountUserMessage,
+} from "../helpers/dbHelper.js"
+
+const sendCount = async (message, client) => {
+  const authorId = message.author.id;
+  const db = client.db;
+
+  const user = db.data.messageCount.filter(({ userId }) => userId === authorId);
+  if (user.messageCount % 1000 === 0) await message.reply(`Et hop, 1000 messages de plus ! \
+Cela vous en fait ${user.messageCount}`);
+};
+
+const addTotalCount = (message, client) => {
+  const authorId = message.author.id;
+  const db = client.db;
+  let count = 0;
+  client.channels.cache.forEach(async (channel) => {
+    if (channel.type === 'GUILD_TEXT') {
+      let loop = 0;
+      let lastId = "";
+      do {
+        let foundMessages = 0;
+        if (!loop) {
+          foundMessages = await channel.messages.fetch({ limit: 100 });
+        } else {
+          foundMessages = await channel.messages.fetch({ limit: 100, after: lastId});
```

Comment:
I tried to add a count for every user message in order to initialise the db, but I am not able to make it work properly. Apparently the fetch() does not accept the "after" term.